recon_test_pack/ScatterRun/par_files/correct_projdata.par

```diff
@@ -0,0 +1,15 @@
+correct_projdata Parameters := 
+
+  input file := nema_proj_f1g1d0b0.hs
+
+  output filename := 20170809_NEMA_UCL.n.hdr.STIR
+  ; use data (1) or set to one (0) := 
+
+  ; apply (1) or undo (0) correction := 
+ 
```

Comment:
can't remember what the defaults are here, but if you want to use this to get the normfactors, then it's better to specify this explicitly:
```
 use data (1) or set to one (0) := 0
 apply (1) or undo (0) correction := 
```

Reply:
ok, pls check